mk/ml.rc.in

```diff
@@ -1,5 +1,7 @@
 #include <Windows.h>
 
+#define ML_FILEFLAGS "@ML_FILEFLAGS@"
+

```

Comment:
Would it be simpler to get rid of this and change the two `#if ML_FILEFLAGS` lines below to `#if @ML_FILEFLAGS@`?  Then this variable would be used the same way as all the others.